### ISSUE TITLE
Metrics in Model Creation

### ISSUE THREAD

**Issue by Kushoza8** [aswintechguy/Deep-Learning-Projects#5]
# THIS IS THE LINE

**model.compile(loss=['binary_crossentropy', 'mae'], optimizer='adam', metrics=['accuracy'])**

In recent updates  of keras and tensorflow it is asking for atleast 2 metrics as we are predicting for age and gender.
So Please check accordingly.

**Comment by aswintechguy** [comment]
This is resolved and the code is updated

**Comment by Kushoza8** [comment]
Thank you
Also  I have implemented  several models for age and gender detection. 
Like VGG, Resnet and Efficient Net models.
Can I contribute to your repository with my codes?
